### DISCUSSION TITLE
Talos in Proxmox - No such host

### DISCUSSION THREAD

**Discussion by kenho811** [siderolabs/talos#8568] [Q&A]
I built two VMs in proxmox, using the image here

**Link: https://github.com/siderolabs/talos/releases/download/v1.6.7/metal-amd64.iso -L -o _out/metal-amd64.iso**

I followed the instructions here: https://www.talos.dev/v1.6/talos-guides/install/virtualized-platforms/proxmox/

However, the console suggests that the talos VM cannot pull the image

![image](https://github.com/siderolabs/talos/assets/57944769/f2afd009-b3f8-46e9-b7eb-b6c439b6e968)

I have other VMs using vmbr0 and they all can connect to the internet.

Below is the spec of the two talos VMs.

![image](https://github.com/siderolabs/talos/assets/57944769/63c9614d-7406-448a-9072-9a7fdd75abaf)

![image](https://github.com/siderolabs/talos/assets/57944769/4378c2ed-c3e5-4cc6-9211-e2421a171cec)

**Comment by smira** [comment]
As you can see in the logs, your DNS server specified returns 'no such host'. Either the DNS is wrong, or something else, but I don't see a problem with Talos Linux here so far.
